src/components/NavBar.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import logo from '../assets/banana-01.png';
+import { useNavigate, NavLink } from 'react-router-dom';
+import { AuthContext } from '../context/AuthContext';
+import { useContext } from 'react';
+
+function NavBar() {
+  
+  const navigate = useNavigate();
+
+  // Dit zijn de properties die wij globaal binnenkrijgen via useContext.
+  // isAuth is de toggle true - flase ingelogd of niet
+
+  const { isAuth, logOutFunction, email } = useContext(AuthContext);
+
+  // console.log(isAuth);
+
+  return (
+    <nav>
+        <NavLink to="/">
+          <span className="logo-container">
+            <img src={logo} alt="logo"/>
+            <h3>
+              Banana Security
+            </h3>
+          </span>
+        </NavLink>
+
+      <div>
+        { isAuth === false
+
+        ? <>         
+        <button
+          type="button"
+          onClick={() => navigate('/signin')}
+        >
+          Log in
+        </button>
+        <button
+          type="button"
+          onClick={() => navigate('/signup')}
+        >
+          Registreren
+        </button>
+        </>
+        :
+
+        <>
+
+        <p>Ingelogd als: { email }</p>
+
+        <button
+          type="button"
+          onClick={ logOutFunction }
+          
+        >
+          Log out
+        </button>
+
+        </>
+
+        }
```

Comment:
Probeer wanneer je een ternary operator gebruikt om de boolean expressie makkelijk leesbaar te houden. Nu valt `isAuth === false` wel mee (en had je dus eigenlijk `isAuth` willen hebben). Je gaat in je programmeercarrière vast nog wel een langere expressie maken met meer variabelen, en zodra die meer `=== false` gaan krijgen met nog `||` en/of `&&` operators, dan is het moeilijk te lezen